lib/lifecycles/bump.js

```diff
@@ -141,7 +141,16 @@ function bumpVersion (releaseAs, args) {
         path: args.path
       }, function (err, release) {
         if (err) return reject(err)
-        else return resolve(release)
+        else if (args.zeroMode) {
+          switch (release.releaseType) {
```

Comment:
This is technically not correct :) v0 is a bit complicated.

Reply:
Sorry I'm not clear what you are saying is wrong?

Reply:
EG: there could be a breaking change from 0.0.1 to 0.0.2

Reply:
Ahh true.  I hadn't considered `0.0.x` at all, I always just start projects at `0.1.0` as the way npm handles `0.0.x` is annoying to me (the caret in `^0.0.1` is useless).